destination.go

```diff
@@ -17,12 +17,7 @@ type Destination struct {
 	config DestinationConfig
 }
 
-type DestinationConfig struct {
-	// Config includes parameters that are the same in the source and destination.
-	Config
-	// DestinationConfigParam must be either yes or no (defaults to yes).
-	DestinationConfigParam string `validate:"inclusion=yes|no" default:"yes"`
-}
+type DestinationConfig struct{}
```

Comment:
I'd imagine we're removing this because it was previously unintentionally added?

Reply:
Yup